config/clusters/2i2c/cluster.yaml

```diff
@@ -1,5 +1,4 @@
 name: 2i2c
-image_repo: "quay.io/2i2c/2i2c-hubs-image"
```

Comment:
@yuvipanda Did this image used to be called `pilot-hub`? I am thinking we may need to i) clean the old repo out of quay.io https://quay.io/organization/2i2c, and ii) update the docs here as that still references `pilot-hub`: https://infrastructure.2i2c.org/en/latest/topic/hub-image.html